src/vcCamera.cpp

```diff
@@ -137,46 +137,24 @@ void vcCamera_UpdateMatrices(vcCamera *pCamera, const vcCameraSettings &settings
   pCamera->matrices.projectionNear = udDouble4x4::perspectiveZO(fov, aspect, 0.5f, 10000.f);
 #endif
 
+  *(double*)(&settings.orthographicSize) = pCamera->position.z * udTan(settings.fieldOfView / 2.0);
```

Comment:
Is `orthographicSize` even used anywhere else in the program anymore? Remove it from `vcCameraSettings` and just calculate and use it locally.